components/homepage/howItWorks.tsx

```diff
@@ -0,0 +1,137 @@
+              </div>
+              <div className="text-gray-500 mt-4 text-lg">
+                Create your organization and invite your team members to join.
+                If you are your own organization, create an organization to
+                manage your grants or applications. Organizations receiving
+                funding can apply to grants and those providing funding can
+                create grants.
+              </div>
+            </div>
+            {/* Image */}
+            <div className="bg-darkmaroon pt-8 pl-8 rounded-lg overflow-hidden border border-darkmaroon">
+              <div className="2xl:w-[40rem] w-[36rem] 2xl:h-[25rem] h-[23rem] relative rounded-tl-lg overflow-hidden">
+                <Image
+                  src="/assets/homepage/organization.png"
+                  alt="organization"
+                  fill={true}
+                />
+              </div>
+            </div>
+          </section>
+          {/* Grant section */}
+          <section className="flex items-center justify-between">
+            {/* Image */}
+            <div className="bg-darkmaroon pt-8 pl-8 pb-8 rounded-lg overflow-hidden border border-darkmaroon">
+              <div className="2xl:w-[40rem] w-[36rem] 2xl:h-[25rem] h-[23rem] relative rounded-tl-lg rounded-bl-lg overflow-hidden">
+                <Image
+                  src="/assets/homepage/grant.png"
+                  alt="grant"
+                  fill={true}
+                />
+              </div>
+            </div>
+            {/* Text */}
+            <div className="w-[30rem]">
+              <div className="2xl:text-4xl text-3xl font-semibold">
+                Apply to or create a <p className="text-maroon inline">grant</p>
+              </div>
+              <div className="text-gray-500 mt-4 text-lg">
+                Create grants for non-profits to apply to if you are a donor
+                organization. If you are a non-profit, apply to grants to
+                receive funding. Grants can be created for specific causes or
+                general funding.
+              </div>
+            </div>
+          </section>
+          {/* Application section */}
+          <section className="flex items-center justify-between">
+            {/* Text */}
+            <div className="w-[30rem]">
+              <div className="2xl:text-4xl text-3xl font-semibold">
+                Review <p className="text-maroon inline">applications</p> and
+                choose a <p className="text-maroon inline">recipient</p>
+              </div>
+              <div className="text-gray-500 mt-4 text-lg">
+                Review applications submitted by non-profits and choose a
+                recipient for your grant. After choosing a recipient you will be
+                able to create an appointment time to set up communications with
+                the non-profit organization on how to proceed. Non-profits can
```

Comment:
Not sure if that's the direction we're going (creating appointments to communicate)